guides/v2.3/cloud/project/project-conf-files_services-elastic.md

```diff
@@ -14,13 +14,13 @@ functional_areas:
 *   Supports stop words and synonyms
 *   Indexing does not impact customers until the reindex operation completes
 
-{{site.data.var.ee}} supports Elasticsearch version 5.2.
+{{site.data.var.ee}} supports Elasticsearch version 5.2 and 6.x (requires v2.3.1 or later).
 
 {:.bs-callout .bs-callout-info}
 Elasticsearch 5.2 is only available for 2.2.3 and later. If you are upgrading to {{site.data.var.ee}} 2.1.3, you must change your configuration as discussed in [the 2.1.3 Release Notes]({{ site.baseurl }}/guides/v2.1/cloud/release-notes/CloudReleaseNotes2.1.3.html#cloud-rn-213-es). See [Elasticsearch information]({{ page.baseurl }}/config-guide/elasticsearch/es-overview.html) to learn more.
```

Comment:
Like you mentioned for ES 5.2m do you think we should mention ES6.X is available only for 2.3.1 or later ?

Reply:
> Like you mentioned for ES 5.2 do you think we should mention ES6.X is available only for 2.3.1 or later ?

Since this is the v2.3 topic, I didn't mention it.  Have opened a ticket on the DevDocs side to provide better info about Elasticsearch version support that we can point to.